src/jobs/runAllJobs.ts

```diff
@@ -0,0 +1,51 @@
+import { Identity } from '@overture-stack/ego-token-middleware';
+import moment from 'moment';
+import { Transporter } from 'nodemailer';
+import SMTPTransport from 'nodemailer/lib/smtp-transport';
+
+import logger from '../logger';
+import attestationRequiredNotification from './attestationRequiredNotification';
+import runPauseAppsCheck from './pauseAppCheck';
+import { Report } from './types';
+import { getEmptyReport } from './utils';
+
+const JOB_NAME = 'ALL BATCH JOBS';
+
+export default async function (
+  emailClient: Transporter<SMTPTransport.SentMessageInfo>,
+  user: Identity,
+) {
+  logger.info(`${JOB_NAME} - Initiating...`);
+  // define currentDate here so each job has the same reference date
+  const currentDate = moment.utc().toDate();
+
+  try {
+    const attestationNotificationReport = await attestationRequiredNotification(
+      currentDate,
+      emailClient,
+    );
+    const pausedAppReport = await runPauseAppsCheck(currentDate, emailClient, user);
+
+    // define report to collect all affected appIds
+    // each job will return its own report
+    // this function will build a complete summary
+    // will simply log the summary for now until we decide what to do with it
+    logger.info(`${JOB_NAME} - Completed, generating report.`);
+    const completeReport: Report = {
```

Comment:
This report is going to want all the details i reccomended in the ReportItem comment (job name, startedAt, etc.)